package.json

```diff
@@ -97,14 +97,7 @@
     "@shopify/js-uploader": "github:shopify/js-uploader",
     "@shopify/react-testing": "^1.5.3",
     "@shopify/sewing-kit": "0.88.0",
-    "@storybook/addon-a11y": "5.1.0-rc.0",
```

Comment:
Looks like these might have accidentally be moved to dependencies, I'm going to follow up to Kaelig.